app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <% title _("Statistics"), _('Statistics') %>
-<div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
-</div>
+
+<div id="statistics"></div>
+
+<%= javascript_tag defer: 'defer' do -%>
```

Comment:
another minor nitpick: I believe that webpack doesnt load the js async by default, so this probably have little effect?